src/ConfigValidator.php

```diff
@@ -134,21 +134,20 @@ private function validateTransformationV2Config(): void
         $component = new Components($this->client);
         try {
             $configData = $component->getConfigurationVersion($componentId, $configId, $versionId);
-            var_dump($configData);
         } catch (ClientException $e) {
             throw new InvalidInputException(
                 'Failed to get ' . $componentId . ' configuration: ' . $e->getMessage(),
                 self::CONFIGURATION_INVALID
             );
         }
-        if (empty($configData)) {
-            throw new InvalidInputException('Configuration Row not found.', self::CONFIGURATION_INVALID);
+        if (empty($configData['configuration'])) {
+            throw new InvalidInputException('Configuration not found.', self::CONFIGURATION_INVALID);
         }
         $processor = new Processor();
         try {
             $configData = $processor->processConfiguration(
                 new TransformationV2Config(),
-                ['configuration' => $configData]
+                ['configuration' => $configData['configuration']]
```

Comment:
Um, and this was a bug, right? I just now can't evaluate if there is supposed to be just the `configuration` or whole $configData. ;o)

Reply:
yes, this was the bug that I posted on the slack

Reply:
ok 😄